arch/xtensa/src/common/xtensa_context.S

```diff
@@ -80,32 +81,28 @@
  *
  * Description:
  *
- *   NOTE: MUST BE CALLED ONLY BY 'CALL0' INSTRUCTION!
+ *	 NOTE: MUST BE CALLED ONLY BY 'CALL0' INSTRUCTION!
```

Comment:
Please revert to spaces in comments in this file as well

Reply:
Done.